### PR TITLE
Fix lint in config, dispatcher

### PR DESCRIPTION
**Pull request by kim201212**
### 1. Type of change

Please delete options that are not relevant.

- [ ] New feature
- [ ] Enhancement
- [ ] Bug/fix (non-breaking change which fixes an issue)
- [ ] others (anything other than above)

---

**Related:** 

#344

**Summary**

Fix lint in config, dispatcher

---

### CODE REVIEW

**Comment by kim201212** [comment]
```shell

$ make coverage
?       github.com/dsrvlabs/vatz        [no test files]
# github.com/dsrvlabs/vatz/manager/plugin.test
ld: warning: -no_pie is deprecated when targeting new OS versions
ld: warning: non-standard -pagezero_size is deprecated when targeting macOS 13.0 or later
# github.com/dsrvlabs/vatz/cmd.test
ld: warning: -no_pie is deprecated when targeting new OS versions
ld: warning: non-standard -pagezero_size is deprecated when targeting macOS 13.0 or later
ok      github.com/dsrvlabs/vatz/cmd    7.287s  coverage: 20.2% of statements
ok      github.com/dsrvlabs/vatz/manager/api    1.071s  coverage: 0.0% of statements [no tests to run]
ok      github.com/dsrvlabs/vatz/manager/config 0.533s  coverage: 77.8% of statements
ok      github.com/dsrvlabs/vatz/manager/dispatcher     2.426s  coverage: 7.2% of statements
ok      github.com/dsrvlabs/vatz/manager/executor       1.389s  coverage: 68.9% of statements
ok      github.com/dsrvlabs/vatz/manager/healthcheck    2.069s  coverage: 46.5% of statements
ok      github.com/dsrvlabs/vatz/manager/plugin 15.692s coverage: 51.1% of statements
?       github.com/dsrvlabs/vatz/manager/types  [no test files]
?       github.com/dsrvlabs/vatz/mocks  [no test files]
ok      github.com/dsrvlabs/vatz/monitoring/prometheus  1.705s  coverage: 0.0% of statements
ok      github.com/dsrvlabs/vatz/rpc    3.818s  coverage: 67.2% of statements
ok      github.com/dsrvlabs/vatz/utils  0.692s  coverage: 7.1% of statements

```

**Comment by xellos00** [comment]
@meetrick 
@gnongs 
@heejin-github 
@skonhwang 
Can you review this?

File: manager/dispatcher/pagerduty.go [hunk 1]
```diff
@@ -3,14 +3,16 @@ package dispatcher
...
+
 	pd "github.com/PagerDuty/go-pagerduty"
 	pb "github.com/dsrvlabs/vatz-proto/plugin/v1"
 	tp "github.com/dsrvlabs/vatz/manager/types"
 	"github.com/rs/zerolog/log"
-	"sync"
-	"time"
 )
 
+// SUCCESS is
```
Comment (rootwarp) [inline manager/dispatcher/pagerduty.go:15]: Uh?

File: manager/types/dispatcher.go [hunk 2]
```diff
@@ -19,25 +20,30 @@ type ReqMsg struct {
...
 
+// DiscordMsg is type for sending messages to a discord.
 type DiscordMsg struct {
 	Username  string  `json:"username,omitempty"`
 	AvatarURL string  `json:"avatar_url,omitempty"`
 	Content   string  `json:"content,omitempty"`
 	Embeds    []Embed `json:"embeds"`
 }
 
+// Embed is imformation for detail message.
```
Comment (gnongs) [inline manager/types/dispatcher.go:46]: @kim201212 Please fixed typo

File: manager/types/dispatcher.go [hunk 3]
```diff
@@ -93,17 +103,20 @@ type NotifyInfo struct {
...
 type Channel string
 
+// the type of channel.
 const (
 	Discord   Channel = "DISCORD"
 	Telegram  Channel = "TELEGRAM"
 	PagerDuty Channel = "PAGERDUTY"
 )
 
+// Reminder is for reminnig alert
```
Comment (gnongs) [inline manager/types/dispatcher.go:116]: @kim201212 Don't it need to add `.`?

Comment (kim201212) [reply manager/types/dispatcher.go]: @gnongs I fixed it!

[fix annotation](https://github.com/dsrvlabs/vatz/pull/461/commits/852f05d84f854cf341508d46f7dd7c3e2d19e4e2)

File: manager/types/dispatcher.go (hunk 2 above)
Comment (kim201212) [reply manager/types/dispatcher.go]: Fix the typo
https://github.com/dsrvlabs/vatz/pull/461/commits/3ee0c0617c951831cd45c64b7780d9cc644ba416

**Review by heejin-github — APPROVED** [review]
LGTM

**Review by gnongs — APPROVED** [review]
LGTM!

**Review by rootwarp — APPROVED** [review]
LGTM

**Pull request merged**